cloud-optimized-geotiffs/cogs-details.qmd

```diff
@@ -1,28 +1,55 @@
 
-GeoTIFF format supports many data types. The key is that all bands must be of the same data type. Unlike some other formats you can not mix and match integers (whole numbers) and floats (decimal numbers) in the same file. If you have this use case consider splitting files by data type and using a catalog like STAC to keep track of them, or look at other formats like [Zarr](../zarr/intro.qmd).
+The GeoTIFF format supports many data types. The key is that all bands must be of the same data type. Unlike some other formats, you cannot mix and match integer (whole number) and float (decimal number) data types in the same file. If you have this use case consider splitting files by data type and using a catalog like STAC to keep track of them, or look at other formats like [Zarr](../zarr/intro.qmd).
 
-Scenario: If the COG is intended only for visualization, conversion to 3 band byte will improve performance. 
+Scenario: If the COG is intended only for visualization, conversion to 3 band byte will improve performance.
 
 > GDAL supported Data Types [list](https://gdal.org/drivers/raster/gtiff.html#gtiff-geotiff-file-format)
 
 
-## Compression (aka File Size)
+## Compression
 
-The biggest benefit to compression is on the storage side. It’s always recommended to use a lossless compression method. **Deflate** or **LZW** are the most recommended compression algorithms, there are some choices that depend on the data type and distribution, and if the goal is maximum compression or not. Maximum compression does result in some performance loss.
+The compression used in a GeoTIFF is the greatest determinator of the file's size. If storing a large amount of data, the right compression choice can lead to much smaller file sizes on disk, leading to lower costs.
+
+### Compression codecs
+
+There are a variety of compression codecs supported by GeoTIFF. Compression codecs tend to be split into two camps: _lossy_ compression where the exact original values cannot be recovered or _lossless_ compression which does not lose any information through the compression and decompression process. For most cases, a lossless compression is recommended, but in some cases a lossy compression can be useful and lead to smaller file sizes, such as if the COG is intended to be used only for visualization.
+
+**Deflate** or **LZW** are both **lossless** compression codecs and are the recommended algorithms for general use.
+
+**JPEG** is a **lossy** compression codec useful for true-color GeoTIFFs intended to be used only for visualization. Because it's lossy, it tends to produce smaller file sizes than deflate or LZW. JPEG should only be used with RGB `Byte` data.
```

Comment:
Do we have to mention the whole YCbr color space trick too?

Reply:
I'm not familiar with it, so I'll leave it out and let someone more knowledgeable on that update as relevant 🙂